app/code/Magento/Cms/Controller/Index/Index.php

```diff
@@ -14,33 +23,46 @@ class Index extends \Magento\Framework\App\Action\Action
     protected $resultForwardFactory;
 
     /**
-     * @param \Magento\Framework\App\Action\Context $context
-     * @param \Magento\Framework\Controller\Result\ForwardFactory $resultForwardFactory
+     * @var \Magento\Framework\App\Config\ScopeConfigInterface
+     */
+    protected $scopeConfig;
```

Comment:
Please, try to avoid the usage of protected properties, make it private.

Reply:
Chanched to private.
Why do I need to use private instead of protected?

Reply:
We don't recommend to use inheritance, that why all class properties should be private. The composition is a better way to extend any logic. More information you can find in [Magento Technical Guidelines](https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html)